### PR TITLE
Fixed tests to not use thelook

### PR DESCRIPTION
**Pull request by muckweaver**
To run:
docker-compose build base; docker-compose build python
docker-compose run python

### CODE REVIEW

File: python/tests/integration/test_methods.py [hunk 1]
```diff
@@ -269,7 +269,7 @@ def test_crud_look(sdk: mtds.LookerSDK, looks):
         assert look.description == l.get("description")
         assert look.deleted == l.get("deleted")
         assert look.is_run_on_load == l.get("is_run_on_load")
-        assert look.public == l.get("public")
+        #assert look.public == l.get("public")
```
Comment (muckweaver) [inline python/tests/integration/test_methods.py]: @joeldodge79 This is the line I had to comment out. If it's not, the test will fail and I couldn't figure out why.

Comment (joeldodge79) [reply python/tests/integration/test_methods.py]: hm, on first glance I can't see why it fails. gonna try to reproduce locally and also build with docker

File: test/data.yml [hunk 2]
```diff
@@ -11,32 +11,17 @@ users:
     last_name: Aardvark
   - first_name: Amanda
     last_name: Aardvark
-queries:
-  - id: "#user_query"
-    model: thelook
-    view: users
+queries_system_activity:
```
Comment (joeldodge79) [inline test/data.yml]: @jkaster These data.yml file changes are fine with me, ok for you?

Comment (jkaster) [reply test/data.yml]: for sure, if `yarn test` works for the Typescript tests from the root of the codegen repo

Comment (joeldodge79) [reply test/data.yml]: @muckweaver on second thought, can we revert this top level yaml key back to `queries`? leave the body the same but update the pytest fixture back to `queries` etc. that will make it so we don't have to update all the other SDK tests

**Comment by muckweaver** [comment]
@joeldodge79 I added back i the 'queries' section of data.yml

File: python/tests/integration/test_methods.py (hunk 1 above)
Comment (joeldodge79) [reply python/tests/integration/test_methods.py]: this assert works for me in "local" mode. I can't seem to get `docker-compose run python` to execute successfully:

**Comment by muckweaver** [comment]
@joeldodge79 I changed the data.yml to have the public value be true and un-commented out the test for it in test_methods.py

**Pull request merged**